src/amo/components/Collection/index.js

```diff
@@ -172,13 +175,9 @@ export class CollectionBase extends React.Component<Props> {
 
     if (errorHandler.hasError()) {
       log.warn('Captured API Error:', errorHandler.capturedError);
-      // 401 and 403 are for an add-on lookup is made to look like a 404 on
-      // purpose. See: https://github.com/mozilla/addons-frontend/issues/3061.
-      if (errorHandler.capturedError.responseStatusCode === 401 ||
```

Comment:
yes, there was no real use case except handling the errors.

Reply:
Yeah but did someone explicitly request for 401/403 responses from the collections API to be rendered as 404s? I hope not. I don't see why we would need to do that because the behavior is different from the add-on scenario.

Reply:
> Yeah but did someone explicitly request for 401/403 responses from the collections API to be rendered as 404s? I hope not.

No I don't think so.

Reply:
FWIW: There's more info in the linked issue: https://github.com/mozilla/addons-frontend/issues/3061